src/components/VerificationChain/Arrow.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+interface ArrowProps {
+  direction?: "up" | "down" | "left" | "right";
+  responsiveDirection?: {
+    mobile?: "up" | "down" | "left" | "right";
+    desktop?: "up" | "down" | "left" | "right";
+  };
+  className?: string;
+}
+
+export const Arrow: React.FC<ArrowProps> = ({
+  direction = "down",
+  responsiveDirection,
+  className = "",
+}) => {
+  const getPathsForDirection = (dir: "up" | "down" | "left" | "right") => {
```

Comment:
```suggestion
  const getPathsForDirection = (dir: Direction) => {
```

... once you add the type alias.